### PR TITLE
BL-2463: do not crash if a large allowed words file is selected.

### PR DESCRIPTION
**Pull request by phillip-hopper**

### CODE REVIEW

File: src/BloomBrowserUI/bookEdit/js/interIframeChannel.ts [hunk 1]
```diff
@@ -129,7 +129,11 @@ class interIframeChannel {
...
   }
 
-    getPageWindow(): Window {
+  getPageWindow(): Window {
     return (<HTMLIFrameElement>document.getElementById('page')).contentWindow;
   }
+
+  getAccordionWindow(): Window {
+    return (<HTMLIFrameElement>document.getElementById('accordion')).contentWindow;
+  }
```
Comment (StephenMcConnel) [inline src/BloomBrowserUI/bookEdit/js/interIframeChannel.ts]: This method looks like the only real change in the file.  Is this right?

Comment (phillip-hopper) [inline src/BloomBrowserUI/bookEdit/js/interIframeChannel.ts]: Correct

File: src/BloomBrowserUI/bookEdit/js/readerTools.js [hunk 2]
```diff
@@ -265,30 +268,38 @@ function loadExternalLink(url) {
  * We need to check the size of the decodable reader tool pane periodically so we can adjust the height of the word list
  * @global {number} previousHeight
  */
-function resizeWordList() {
+function resizeWordList(startTimeout) {
+    if (startTimeout === void 0) { startTimeout = true; }
```
Comment (StephenMcConnel) [inline src/BloomBrowserUI/bookEdit/js/readerTools.js]: I assume this is giving startTimeout a default value of true, allowing the method to be called with or without an argument.

File: src/BloomBrowserUI/bookEdit/js/readerTools.ts [hunk 3]
```diff
@@ -339,34 +343,42 @@ function loadExternalLink(url: string): void {
  * We need to check the size of the decodable reader tool pane periodically so we can adjust the height of the word list
  * @global {number} previousHeight
  */
-function resizeWordList(): void {
+function resizeWordList(startTimeout: boolean = true): void {
```
Comment (StephenMcConnel) [inline src/BloomBrowserUI/bookEdit/js/readerTools.ts]: This looks like another way to provide a default value for an input argument.

**Comment by StephenMcConnel** [comment]
Generally looks okay to me, but I'm not sure that I understand TypeScript well enough yet to give an official okay.

File: src/BloomBrowserUI/bookEdit/js/readerTools.ts (hunk 3 above)
Comment (JohnThomson) [inline src/BloomBrowserUI/bookEdit/js/readerTools.ts]: I think that's right...the js being automatically generated from this.

**Pull request merged**